packages/devextreme-cli/commands/themebuider.js

```diff
@@ -123,14 +123,24 @@ const installThemeBuilder = async version => {
     }
 };
 
-const getDevExtremeVersion = () => {
-    const lockFileName = path.join(process.cwd(), 'package-lock.json');
-    const installedDevExtremePackageJson = path.join(process.cwd(), 'node_modules', 'devextreme', 'package.json');
+const getDevextremeInfo = (dependencies) => {
+    const keyValue = Object.keys(dependencies).filter((key) => {
+        return /devextreme@/.test(key);
+    })[0];
```

Comment:
```suggestion
    const keyValue = Object.keys(dependencies).filter((key) => /devextreme@/.test(key))[0];
```

Reply:
It looks like the `some` method suits better

Reply:
Some method returns true or false but we need to get a key. Because yarn.lock has the key with a version like devexteme@latest.